tests/integration/examples/experiments/tc.yaml

```diff
@@ -30,3 +30,6 @@ pipeline:
       params: !@ b1[model].trainable_params
     max_steps: 2
     iter_per_step: 1
+    extra_validation_metrics:
```

Comment:
How does this example test this case? According to the description this is for lists of links right?